vncviewer/UserDialog.cxx

```diff
@@ -164,6 +187,13 @@ void UserDialog::getUserPasswd(bool secure, char** user, char** password)
 
   y += 5;
 
+  keepPasswdCheckbox = new Fl_Check_Button(70, y, win->w()-70-10, 20,
+					   _("Keep password for reconnect"));
+  keepPasswdCheckbox->callback(keepPassword, this);
+  keepPasswdCheckbox->value(keepPasswd_);
```

Comment:
Reconnect is optional, so should we perhaps hide this if reconnect isn't enabled?